src/components/Single-review.jsx

```diff
@@ -1,4 +1,34 @@
+import { useEffect, useState } from "react";
+import * as api from "../api.js";
```

Comment:
This works but as more of a nicety it might be worth destructuring the function you want here rarther than importing everything